backend/test/data_access_objects/test_app_configuration.py

```diff
@@ -62,7 +62,7 @@ def generate_test_config(config_scope: str, version_id: int, is_project: bool) -
         "changedBy": "Tester",
         "createdAt": mock_time,
         "configuration": {
-            "DisabledInstanceTypes": {
+            "EnabledInstanceTypes": {
```

Comment:
Are we not testing this yet? Confusing how this passes with no other changes. I would expect conditionals to need to be inverted.